t/FHEM/14_SD_UT/testData.json

```diff
@@ -2417,5 +2417,54 @@
       "id" : 130,
       "module" : "SD_UT",
       "name" : "CREATE_6601TL"
+   },
+   {
+      "data" : [
+         {
+            "comment" : "Remote control with two buttons for Halemeier LED controllers (typ HA-HX2 for actor HA-RX-M2-1)",
+            "dmsg" : "P132#01EFAC",
+            "internals" : {
+               "DEF" : "HA_HX2 01EF",
+               "NAME" : "HA_HX2_01EF"
+            },
+            "readings" : {
+               "deviceCode" : "01EF",
+               "state" : "off"
+            },
+            "rmsg" : "MU;P0=307;P1=-685;P2=-350;P3=658;P4=-12510;D=01010102310101010102310231010232340232323232323231010101023101010101023102323232323232323101010102310101010102310231010232340232323232323231010101023101010101023102310102323402323232323232310101010231010101010231023101023234023232323232323101010102310101;CP=0;R=232;O;",
+            "tests" : [
+               {
+                  "attributes" : {
+                     "model" : "HA_HX2"
+                  },
+                  "comment" : "#0 state off"
+               }
+            ]
+         },
+         {
+            "comment" : "Remote control with two buttons for Halemeier LED controllers (typ HA-HX2 for actor HA-RX-M2-1)",
+            "dmsg" : "P132#85EFAA",
+            "internals" : {
+               "DEF" : "HA_HX2 85EF",
+               "NAME" : "HA_HX2_85EF"
+            },
+            "readings" : {
+               "deviceCode" : "85EF",
+               "state" : "on"
+            },
+            "rmsg" : "MU;P0=-696;P1=312;P2=-371;P3=637;P4=-12847;D=01012301230123012341012323232301230101010123010101010123012301230123410123232323012301010101230101010101230123012301234101232323230123010101012301010101012301230123012341012323232301230101010123010101010123012301230123410123232323012301010101230101010101;CP=1;R=236;O;",
+            "tests" : [
```

Comment:
Warum prüfst Du hier keine Readings oder internals?

Reply:
Die Ergänzung der Tests stellt für uns immer etwas „Acker“ dar. Wir setzen sie bestmöglich um :) Wenn du auf etwas bestimmtes testen möchtest, dann hast du freie Hand.

Reply:
Mit Acker meinst Du Arbeit?

War auch erst mal nur eine Frage, da ich den Zweck des Tests nicht verstanden habe.

Reply:
Arbeit eher zweitrangig. Teilweise vermuten wir nur was die Tests machen oder wie die Zeilen zusammenspielen. Das ist manchmal nur mit probieren unsererseits möglich. Aber wir kommen doch soweit gut klar :) Wenn du beim betrachten eine Frage oder Idee hast, kannst du die Vollendung durchführen. (Das ist keinerlei Kritik, wir arbeiten nur sehr selten mit der Art der Tests bzw. Verstehen nur ein Bruchteil des Zusammenspiels in den Testfiles)

Reply:
Supi.
Also die Tests passen, ich musste mich da auch erst noch einmal wieder rein denken, denn ich hatte es so in Erinnerung, dass alles was nicht unterhalb von "Tests" steht gesetzt und nicht geprüft wird.
Das ist aber nicht so, somit werden die Readings im Test verifiziert auch wenn sie außerhalb von "Tests" stehen.
Das habe ich so gemacht um möglichst nahe zum "älteren" Format zu bleiben.